doc/sphinx/user/methods/index.md

```diff
@@ -1,7 +1,10 @@
 (cha:methods)=
-# Geodynamic modeling assumptions and numerical methods in ASPECT
+# Model assumptions and numerical methods
```

Comment:
Modelling means something different to me here.

Reply:
Hmm, I dont think I understand. Do you mean you think "geodynamic modeling assumptions" and "model assumptions" have different meaning and you prefer the original one? Can you elaborate? My main purpose was to shorten this heading while keeping the meaning, because it was very long.

Reply:
I am thinking "Modeling assumptions" is better than "Model assumptions".